File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/globalization-and-localization-of-excel-solutions_2.vb
    Private Sub ConvertDate_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) _
        Handles ConvertDate.Click

        Try
            Dim dbl As Double = Me.Range("A5").Value2
            Dim dt As System.DateTime = System.DateTime.FromOADate(dbl)
            Me.Range("A7").Value2 = dt

        Catch ex As Exception
            MessageBox.Show(ex.Message)
        End Try
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/globalization-and-localization-of-excel-solutions_2.cs
        private void ConvertDate_Click(object sender, EventArgs e)
        {
            try
            {
                double dbl = (double)(this.Range["A5"].Value2);
                System.DateTime dt = System.DateTime.FromOADate(dbl);
                this.Range["A7"].Value2 = dt;
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message);
            }
        }